lambdas/helpers/dynamoDBHelper.js

```diff
@@ -2,40 +2,79 @@
 const AWS = require('aws-sdk');
 const log = require('./logger');
 
+const testConfig = {
+  appEnv: 'test',
+  domain: 'sites-test.federalist.18f.gov',
+  tableName: 'federalist-proxy-test',
+  siteKey: 'id',
+};
+
+const stagingConfig = {
+  appEnv: 'staging',
+  domain: 'sites-staging.federalist.18f.gov',
+  tableName: 'federalist-proxy-staging',
+  siteKey: 'id',
+};
+
+const prodConfig = {
+  appEnv: 'production',
+  domain: 'sites-prod.federalist.18f.gov',
+  tableName: 'federalist-proxy-prod',
+  siteKey: 'id',
+};
+
 const getSiteConfig = async (params) => {
   const docClient = new AWS.DynamoDB.DocumentClient({
     httpOptions: { connectTimeout: 120000, timeout: 120000 },
   });
-  return docClient.query(params)
+  return docClient.get(params)
     .promise()
     .catch((err) => {
       throw new Error(`Unable to query. Error: ${JSON.stringify(err, null, 2)}`);
     })
-    .then(({ Count, Items }) => {
-      if (Count > 0) {
-        const [item] = Items;
-        log(`\nQuery succeeded: item found @id:${JSON.stringify(item.id)}\n`);
-        return item.settings;
+    .then(({ Item }) => {
+      if (Item) {
+        log(`\nQuery succeeded: item found @id:${JSON.stringify(Item.id)}\n`);
+        return Item.settings;
       }
       log('\nQuery succeeded: no results found!!\n');
       return undefined;
     });
 };
 
-const getSiteQueryParams = (tableName, siteKey, siteKeyValue) => {
-  const expressionAttributeNames = {
-    [`#${siteKey}`]: siteKey,
-  };
+const validateProxyFunctionName = functionName => /^us-east-1:federalist-proxy-(prod|staging|test)-(viewer|origin)-(request|response):\d+$/
+  .test(functionName);
 
-  const expressionAttributeValues = {
-    [`:${siteKey}`]: siteKeyValue,
-  };
+const getAppConfig = (functionName) => {
+  let appConfig = {};
+  if (validateProxyFunctionName(functionName)) {
+    if (/staging/.test(functionName)) {
+      appConfig = stagingConfig;
+    } else if (/prod/.test(functionName)) {
+      appConfig = prodConfig;
+    } else if (/test/.test(functionName)) {
+      appConfig = testConfig;
+    }
+  }
+  return appConfig;
+};
 
+const getSiteKeyValue = (host, appDomain) => {
```

Comment:
It looks like this function strips the domain from the end of the host if it is present or returns undefined.

Wondering if:
1. Renaming this function something like `stripDomain` would be more semantic.
2. It should throw an error if the host doesn't end in the domain OR not even check and let it not find an entry in the db. Does it error right now if it looks up a records with a key === `undefined` in Dynamo?